services/management/backup/locations_service_test.go

```diff
@@ -169,3 +169,57 @@ func TestListBackupLocations(t *testing.T) {
 	})
 
 }
+
+func TestRemoveBackupLocation(t *testing.T) {
+	ctx := context.Background()
+	sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+	db := reform.NewDB(sqlDB, postgresql.Dialect, reform.NewPrintfLogger(t.Logf))
+
+	svc := NewLocationsService(db)
+
+	req := &backupv1beta1.AddLocationRequest{
+		Name: gofakeit.Name(),
+		PmmClientConfig: &backupv1beta1.PMMClientLocationConfig{
+			Path: "/tmp",
+		},
+	}
+	res1, err := svc.AddLocation(ctx, req)
+	require.NoError(t, err)
+	req.Name = gofakeit.Name()
+	_, err = svc.AddLocation(ctx, req)
+	require.NoError(t, err)
+	req.Name = gofakeit.Name()
+	res3, err := svc.AddLocation(ctx, req)
+	require.NoError(t, err)
+	req.Name = gofakeit.Name()
+
+	assertNotFound := func(id string, locations []*backupv1beta1.Location) func() bool {
+		return func() bool {
+			found := false
+			for _, loc := range locations {
+				if loc.LocationId == id {
+					found = true
+					break
+				}
+			}
+			return !found
```

Comment:
I think the _found_ variable can be deleted. Instead of _break_ you can _return false_, and at the end you can _return true_.